client/src/main/java/com/alibaba/nacos/client/naming/net/HttpClient.java

```diff
@@ -16,18 +16,26 @@
 package com.alibaba.nacos.client.naming.net;
 
 import com.alibaba.nacos.api.common.Constants;
+import com.alibaba.nacos.client.config.impl.HttpSimpleClient;
 import com.alibaba.nacos.common.utils.HttpMethod;
 import com.alibaba.nacos.common.utils.IoUtils;
 import com.google.common.net.HttpHeaders;
 import org.apache.commons.lang3.StringUtils;
```

Comment:
In nacos client, I have already removed the dependency of apache-common-lang3, you can merge the code from the develop branch, thanks.